main.go

```diff
@@ -191,6 +191,7 @@ The log file is stored in $BITRISE_DEPLOY_DIR, and its full path is available in
 	args = append(args, "build-for-testing")
 
 	cmd := command.New(args[0], args[1:]...)
+	log.Donef("=> ", cmd.PrintableCommandArgs())
```

Comment:
we have a convention for outputting commands:
```
$ my command here
```

also no string formatter in: `"=> "`

so it would become sometinh like 
`"$ %s"`